resources/migrations/001_update_migrations.yaml

```
@@ -4365,6 +4365,21 @@ databaseChangeLog:
                     nullable: true
                   remarks: 'If true, the table requires a filter to be able to query it'
 
+  - changeSet:
+      id: v49.00-012
+      author: qnkhuat
+      comment: Add metabase_field.database_partitioned
+      changes:
+        - addColumn:
+            tableName: metabase_field
+            columns:
+              - column:
+                  name: database_partitioned
+                  type: boolean
+                  constraints:
+                    nullable: true
+                  remarks: 'Whether the field is a partitioned field'
```

Technically fields aren't partitioned, tables are
```suggestion
                  remarks: 'Whether the table is partitioned by this field'
```